From ed0624ab614aeac7a0d64dae4ae0dd3b4a4d895d Mon Sep 17 00:00:00 2001
From: Dave Reisner <dreisner@archlinux.org>
Date: Tue, 14 Mar 2017 10:50:26 -0400
Subject: [PATCH 2/2] avoid erroring when a builtin-module can't be found

This is harmless fallout from 5249350, which tries harder to re-add
aliases of builtins.
---
 functions | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/functions b/functions
index 7288c44..8130b63 100644
--- a/functions
+++ b/functions
@@ -390,7 +390,7 @@ add_module() {
     done < <(modinfo -b "$_optmoduleroot" -k "$KERNELVERSION" -0 "$target" 2>/dev/null)
 
     if (( !found )); then
-        (( ign_errors )) && return 0
+        (( ign_errors || _addedmodules["$target"] )) && return 0
         error "module not found: \`%s'" "$target"
         return 1
     fi
-- 
2.14.1

